source/includes/examples/_simpy-elevator.html.md

```diff
@@ -0,0 +1,200 @@
+# SimPy Elevator Simulation
+
+> ![ASCII Elevator Output](../images/elevator_output.png)
```

Comment:
Yay for recapture!  some description might help too - 

the elevator logs output every 100 seconds, then shows the state of the world, and then a list of recent passengers.
the world state is the floor, the number of people waiting, plus the elevator, and the number of people inside. so

` 1:  0|      2:  1|       3:  0| [_1_]` shows zero people on the first floor, one person waiting on the second floor, and one person in the elevator on the third floor.